File: N/1450-number-of-students-doing-homework-at-a-given-time/main.go
package main

import "fmt"

func busyStudent(startTime []int, endTime []int, queryTime int) int {
	c := 0

	for i := 0; i < len(startTime); i++ {
		if startTime[i] <= queryTime && queryTime <= endTime[i] {
			c++
		}
	}
	return c
}

func main() {
	fmt.Println("a")
}

File: N/1450-number-of-students-doing-homework-at-a-given-time/main.rs
struct Solution {}

impl Solution {
  pub fn busy_student(start_time: Vec<i32>, end_time: Vec<i32>, query_time: i32) -> i32 {
    (0..start_time.len())
      .map(|idx| {
        if start_time[idx] <= query_time && end_time[idx] >= query_time {
          1
        } else {
          0
        }
      })
      .sum::<i32>()
  }
}
